plugin.audio.radiothek/changelog.txt

```diff
@@ -0,0 +1,10 @@
+0.0.2
```

Comment:
please add changelog to your gitattributes file to avoid submitting it again all the time. Changelog was removed in favor of the `<news>` tag in addon.xml